token/cli/src/main.rs

```diff
@@ -5308,6 +5603,125 @@ async fn process_command<'a>(
             )
             .await
         }
+        (CommandName::ConfigureConfidentialTransferAccount, arg_matches) => {
+            let (owner_signer, owner) =
+                config.signer_or_default(arg_matches, "owner", &mut wallet_manager);
+
+            let token_account =
+                config.pubkey_or_default(arg_matches, "account", &mut wallet_manager)?;
+
+            // Deriving ElGamal and AES key from signer. Custom ElGamal and AES keys will be
+            // supported in the future once upgrading to clap-v3.
+            let elgamal_keypair =
+                ElGamalKeypair::new_from_signer(&*owner_signer, &token_account.to_bytes()).unwrap();
+            let aes_key =
+                AeKey::new_from_signer(&*owner_signer, &token_account.to_bytes()).unwrap();
+
+            if config.multisigner_pubkeys.is_empty() {
+                push_signer_with_dedup(owner_signer, &mut bulk_signers);
+            }
+
+            let maximum_credit_counter = value_t!(
+                arg_matches.value_of("maximum_pending_balance_credit_counter"),
+                u64
+            )
+            .ok();
+
+            command_configure_confidential_transfer_account(
+                config,
+                token_account,
+                owner,
+                maximum_credit_counter,
+                &elgamal_keypair,
+                &aes_key,
+                bulk_signers,
+            )
+            .await
+        }
+        (CommandName::EnableConfidentialCredits, arg_matches) => {
```

Comment:
nit: You could probably combine all of these by doing:

```
(c @ CommandName::EnableConfidentialCredits, arg_matches) |
(c @ CommandName::DisableConfidentialCredits, arg_matches) |
...
```

Then

```
let (allow_confidential_credits, allow_non_confidential_credits) = match c {
    CommandName::EnableConfidentialCredits => (Some(true), None),
    ...
};
```

Reply:
Oh I think I never used match bindings before. I learned something new. Thanks!